test/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+#############################################################################
+# Copyright (c) 2023, xeus-cpp contributors                                 #
+#                                                                           #
+# Distributed under the terms of the BSD 3-Clause License.                  #
+#                                                                           #
+# The full license is in the file LICENSE, distributed with this software.  #
+#############################################################################
+
+
+# Unit tests
+# ==========
+
+cmake_minimum_required(VERSION 3.1)
+
+if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+    project(xeus-cpp-test)
+
+    enable_testing()
+
+    find_package(xeus-cpp REQUIRED CONFIG)
+endif()
+
+message(STATUS "Forcing tests build type to Release")
+set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
```

Comment:
Why do we need to force the tests to be in release while everything else has been selected to be in Debug, for example? That might cause build inconsistencies when we have `#ifndef NDEBUG`, etc.

Reply:
That does make sense. We can get rid of it.